### ISSUE TITLE
Deploy

### ISSUE THREAD

**Issue by joakim-hove** [FriskByBergen/python-friskby-controlpanel#9]
Could we deploy a version of this server - running on mock/synthetic backend; so that it was simple for everyone to see the status?

**Comment by pgdr** [comment]
That's a good idea, and the answer is no (now), but should be yes in the not-so-distant future.

I think it depends on something that I would like for us to have in FriskByBergen/python-friskby#17 namely the ability to mock the server and sensor.

**Comment by pgdr** [comment]
But you can of course install it locally (use FriskPITest) with pip (it will also install rpiparticle and friskby) with service, and just go to http://localhost.  The sampler will crash, of course.

I will recommend using VirtualBox + Debian so you don't install the services on your system.

**Comment by pgdr** [comment]
Thanks for raising the issue, Joakim.  I just discovered a bug.

I am doing the following (in Debian running in VirtualBox, highly recommended)

```bash
sudo pip install friskby-controlpanel
sudo pip install rpiparticle
sudo systemctl daemon-reload
sudo systemctl start friskby-sampler
sudo systemctl start friskby-submitter
sudo systemctl start friskby-controlpanel
```
Then go to firefox http://localhost

However, the friskby module (from python-friskby) has an error; the `sensors` submodule hasn't been deployed.

**Comment by pgdr** [comment]
Okay, that was quite helpful.  Thanks for the suggestion, @joakim-hove.

I found several bugs, and we should start fixing them ASAP.

**Comment by pgdr** [comment]
![friskby-controlpanel](https://cloud.githubusercontent.com/assets/17569537/24590443/48f64074-17ed-11e7-877e-7a99a077c848.png)

**Comment by semafor** [comment]
Right, so most of these issues should be solved in 0.4.2 which I will upload right now.

**Comment by semafor** [comment]
@joakim-hove yes, we can do this without too much effort, I think. If you take a look at the code, the server accepts a `FriskbyInterface` which we could swap with something else. In testing, we use a `FakeFriskbyInterface`, and I could certainly create a `SampleDataFriskbyInterface` which would be used if you started the controlpanel from the command line (e.g. `python run_sample_data_controlpanel.py 0.0.0.0 5003`).

However, I'm not entirely sure how we'd edit the sample data as the application was running? The script could accept command line arguments (e.g. `… --uploaded_samples=100 --total_samples=300`) but that hardly scales. Maybe there's a JSON that acts as the _universe_? 

What do you think?
